src/rewriters/CMakeLists.txt

```diff
@@ -2,4 +2,7 @@ add_library(rewriters OBJECT "")
 
 target_sources(rewriters
     PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/DistinctRewriter.cc"
-)
\ No newline at end of file
+)
+
+install(FILES Rewriter.h Substitutor.h ArithmeticEqualityRewriter.h
+    DESTINATION ${INSTALL_HEADERS_DIR})
```

Comment:
Do we need to expose these?
It would be good to give users a nice API to rewrite terms, I am just not sure if we need to expose everything.

Reply:
For now let's not expose them.